x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings_management/index.tsx

```diff
@@ -0,0 +1,262 @@
+  } = useFlyoutModalVisibility();
+  const [deletedConversation, setDeletedConversation] = useState<ConversationTableItem | null>();
+
+  const {
+    isFlyoutOpen: deleteConfirmModalVisibility,
+    openFlyout: openConfirmModal,
+    closeFlyout: closeConfirmModal,
+  } = useFlyoutModalVisibility();
+
+  const onConversationSelectionChange = useConversationChanged({
+    allSystemPrompts,
+    conversationSettings,
+    conversationsSettingsBulkActions,
+    defaultConnector,
+    setConversationSettings,
+    setConversationsSettingsBulkActions,
+    onSelectedConversationChange,
+  });
+
+  const onEditActionClicked = useCallback(
+    (rowItem: ConversationTableItem) => {
+      openEditFlyout();
+      onConversationSelectionChange(rowItem);
+    },
+    [onConversationSelectionChange, openEditFlyout]
+  );
+
+  const onConversationDeleted = useConversationDeleted({
+    conversationSettings,
+    conversationsSettingsBulkActions,
+    setConversationSettings,
+    setConversationsSettingsBulkActions,
+  });
+
+  const onDeleteActionClicked = useCallback(
+    (rowItem: ConversationTableItem) => {
+      setDeletedConversation(rowItem);
+      onConversationDeleted(rowItem.title);
+
+      closeEditFlyout();
+      openConfirmModal();
+    },
+    [closeEditFlyout, onConversationDeleted, openConfirmModal]
+  );
+
+  const onDeleteConfirmed = useCallback(() => {
+    if (Object.keys(conversationsSettingsBulkActions).length === 0) {
+      return;
+    }
+    closeConfirmModal();
+    handleSave(true);
+    setConversationsSettingsBulkActions({});
+  }, [
+    closeConfirmModal,
+    conversationsSettingsBulkActions,
+    handleSave,
+    setConversationsSettingsBulkActions,
+  ]);
+
+  const onDeleteCancelled = useCallback(() => {
+    setDeletedConversation(null);
+    closeConfirmModal();
+    onCancelClick();
+  }, [closeConfirmModal, onCancelClick]);
+
+  const { getConversationsList, getColumns } = useConversationsTable();
+
+  const conversationOptions = getConversationsList({
+    allSystemPrompts,
+    actionTypeRegistry,
+    connectors,
+    conversations: conversationSettings,
+    defaultConnector,
+  });
+
+  const onSaveCancelled = useCallback(() => {
+    closeEditFlyout();
+    onCancelClick();
+  }, [closeEditFlyout, onCancelClick]);
+
+  const onSaveConfirmed = useCallback(() => {
+    closeEditFlyout();
+    handleSave(true);
+    setConversationsSettingsBulkActions({});
+  }, [closeEditFlyout, handleSave, setConversationsSettingsBulkActions]);
+
+  const columns = useMemo(
+    () =>
+      getColumns({
+        conversations: conversationSettings,
+        onDeleteActionClicked,
+        onEditActionClicked,
+        allSystemPrompts,
+      }),
+    [allSystemPrompts, conversationSettings, getColumns, onDeleteActionClicked, onEditActionClicked]
+  );
+
+  const sorting = useMemo(
+    () => ({
+      sort: {
+        field: 'updatedAt',
+        direction: 'desc' as const,
+      },
+    }),
+    []
+  );
```

Comment:
Is it possible to display the sorting arrow on this column? When you edit a Conversation you end up seeing it bounce to the top of the table since it's `updatedAt` changed, but it's not clear the table is sorted by `updatedAt`.

Since this is an `EuiInMemory` table can we just enable sorting for all columns?

<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/cb53d735-12dd-47c0-8e59-a7e4f5bd3e08" />
</p>